src/appleseed/renderer/modeling/material/genericmaterial.cpp

```diff
@@ -101,6 +101,13 @@ namespace
                     "this may lead to unexpected or unphysical results.",
                     context.get());
             }
+            if  (m_render_data.m_volume && has_alpha_map())
```

Comment:
Separate both `if` with a blank line.

Reply:
Thanks, Done.

Reply:
Check `m_render_data.m_alpha_map` instead of using `has_alpha_map()`, as it's done in the EDF+alpha map case. Ultimately it should be the same thing, but there's no reason to be inconsistent.

Reply:
done, thanks.